gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/SpecCatalogListener.java

```diff
@@ -56,18 +57,20 @@ public AddSpecCallback(Spec addedSpec) {
   public static class DeleteSpecCallback extends Callback<SpecCatalogListener, Void> {
     private final URI _deletedSpecURI;
     private final String _deletedSpecVersion;
+    private final Properties _headers;
 
-    public DeleteSpecCallback(URI deletedSpecURI, String deletedSpecVersion) {
+    public DeleteSpecCallback(URI deletedSpecURI, String deletedSpecVersion, Properties headers) {
```

Comment:
Naming should be consistent. `props` above, but `headers` here.

Reply:
done